restful.module

```diff
@@ -561,3 +564,18 @@ function restful_date_time_format_element_validate($element, &$form_state) {
     )));
   }
 }
+
+/**
+ * Implements hook_restful_resource_alter().
+ */
+function restful_restful_resource_alter(ResourceInterface &$resource) {
```

Comment:
> Keep in mind that the composition order is important.

So how do we impose this order easily, without other implementing module needing to deal with ``hook_module_implenmets_alter()``? (if no easier way we can keep it, just wondering if you have an idea)